test/inductor/test_efficient_conv_bn_eval.py

```diff
@@ -0,0 +1,195 @@
+# Owner(s): ["module: inductor"]
+import copy
+import importlib
+import itertools
+import os
+import sys
+import unittest
+
+import torch
+from torch import nn
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+
+from torch._dynamo.test_case import TestCase
+from torch._dynamo.utils import counters
+
+from torch.testing._internal.common_utils import IS_CI, IS_WINDOWS, TEST_WITH_ASAN
+
+from torch.testing._internal.inductor_utils import HAS_CPU, HAS_CUDA
+
+if IS_WINDOWS and IS_CI:
+    sys.stderr.write(
+        "Windows CI does not have necessary dependencies for test_torchinductor yet\n"
+    )
+    if __name__ == "__main__":
+        sys.exit(0)
+    raise unittest.SkipTest("requires sympy/functorch/filelock")
+
+importlib.import_module("functorch")
+importlib.import_module("filelock")
```

Comment:
I don't think this is needed?

Reply:
This is copied from `test/inductor/test_binary_folding.py`. It seems to be related with windows CI. I don't know if it can be safely removed. @XiaobingSuper do you have any idea?